cfme/tests/cli/test_appliance_console.py

```diff
@@ -52,6 +54,19 @@
 ]
 
 
+def is_port_open(hostname, port):
```

Comment:
This should be using `cfme.utils.net.net_check`

https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/net.py#L61

Reply:
Methods like this shouldn't be implemented directly within the test that needs them, as inevitably another test within the framework will need the same functionality.